.github/workflows/daily_perf_tests.yml

```diff
@@ -13,6 +13,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 600
     steps:
+    - name: Define AKS cluster name
+      run: |
+        cluster_name="bpc-daily-perf-cluster"
+        echo "::set-output name=cluster-name::$cluster_name"
+      id: cluster 
```

Comment:
shouldn't we add this to outputs?
Ex  - 
```
    outputs:
      matrix: ${{ steps.write.outputs.test }}
      
```

Reply:
done